jaseci_core/setup.py

```diff
@@ -15,10 +15,14 @@
         "flake8",
         "stripe",
         "pydantic",
+        "prometheus_api_client==0.5.1",
+        "prometheus-client==0.14.1",
+        "kubernetes==23.6.0",
         "docstring-parser",
     ],
     package_data={
-        "": ["*.ini"],
+        "": ["*.ini", "*.yaml"],
```

Comment:
@marsninja This is so that the redis.yaml will be included when the jaseci_core package is installed.